apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/Oppgavelista.tsx

```diff
@@ -30,6 +30,21 @@ export const HeaderPadding = styled.span`
   padding-left: 20px;
 `
 
+// Spesiell håndtering for ferdigstilt status for oppgaver som medfører en ny oppgave til attestant
+export function formaterOppgavestatus(oppgave: OppgaveDTOny): string {
+  if (
+    oppgave.status === 'FERDIGSTILT' &&
+    (oppgave.type === 'FOERSTEGANGSBEHANDLING' ||
+      oppgave.type === 'REVURDERING' ||
+      oppgave.type === 'MANUELT_OPPHOER' ||
+      oppgave.type === 'UNDERKJENT')
+  ) {
+    return 'Til attestering'
```

Comment:
Jeg så diskusjonen på Slack. Litt usikker på om det var helt tydelig for Magny at konsekvensen blir at statusteksten "Til attestering" ikke endrer seg selv etter at saken er ferdig attestert?

Reply:
Hmm, nå second-guesser jeg meg selv. Trippelsjekker med Magny :D

Reply:
Underkjent?

Reply:
Er manuelt opphør en revurdering?

Reply:
Du har helt rett, dette skal ikke inn :D

Reply:
Vi så litt på dette med at oppgavetype="FOERSTEGANGSBEHANDLING",  burde den heller vært "fatting"? hvis revurderinger skal ha en to-trinnskontroll så burde vi kanskje se litt mer på dette